### DISCUSSION TITLE
Rocket v0.5.0-rc.3

### DISCUSSION THREAD

**Discussion by SergioBenitez** [rwf2/Rocket#2500] [Announcements]
See the [CHANGELOG](https://github.com/SergioBenitez/Rocket/blob/v0.5.0-rc.3/CHANGELOG.md#version-050-rc3-mar-23-2023), [news article](https://rocket.rs/v0.5-rc/news/2023-03-23-version-0.5-rc.3/), [migration guide](https://rocket.rs/v0.5-rc/guide/upgrading/), and [FAQ](https://rocket.rs/v0.5-rc/guide/faq/) for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/SergioBenitez/Rocket/releases/tag/v0.5.0-rc.3'>Rocket v0.5.0-rc.3</a>.</em>

**Comment by NexRX** [comment]
My eyes widened with joy seeing this.

**Comment by griffi-gh** [comment]
Yes! Finally a new version!
